app/src/main/java/me/hufman/androidautoidrive/phoneui/fragments/SpotifyApiErrorDialog.kt

```diff
@@ -1,18 +1,76 @@
 package me.hufman.androidautoidrive.phoneui.fragments
 
+import android.app.Activity
 import android.app.AlertDialog
 import android.app.Dialog
+import android.content.Intent
 import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.Button
+import android.widget.TextView
 import androidx.fragment.app.DialogFragment
 import me.hufman.androidautoidrive.R
-import java.lang.IllegalStateException
+import me.hufman.androidautoidrive.phoneui.SpotifyAuthorizationActivity
+import me.hufman.androidautoidrive.phoneui.visible
 
 class SpotifyApiErrorDialog: DialogFragment() {
 	companion object {
+		const val TAG = "SpotifyApiErrorDialog"
 		const val EXTRA_CLASSNAME = "classname"
 		const val EXTRA_MESSAGE = "message"
+		const val EXTRA_WEB_API_AUTHORIZED = "webApiAuthorized"
 	}
+
+	lateinit var webApiMsgTextView: TextView
```

Comment:
`lateinit` makes sense for these, because they only exist after `onCreateDialog` has inflated the layout